src/modules/Home/components/Home/Home.tsx

```diff
@@ -0,0 +1,324 @@
+import octocatSVG from 'assets/images/octocat.svg';
+import openedBookSVG from 'assets/images/openedBook.svg';
+import shieldSVG from 'assets/images/shield.svg';
+import userSVG from 'assets/images/user.svg';
+import Flex from 'common/components/Flex/Flex';
+import MainSlider from 'modules/Home/containers/MainSliderContainer';
+
+import styles from './home.scss';
+
+interface IProps {
+	currentMainSlide: any;
+}
+
+const Home: React.FC<IProps> = props => (
+	<div className={cn(styles.home, styles[props.currentMainSlide.name])}>
+		<Flex justifyContent='center' className={styles.intro}>
+
+			<Flex alignItems='center' direction='column' className={styles.mainTitles}>
+				<LogoIcon fill='#FFFFFF' className={styles.introLogo} />
+				<p className={styles.mainTitles_top}>Introducing CusDeb</p>
+				<p className={styles.mainTitles_bottom}>
+					CusDeb is a powerful service for building customized operating system
+					images for single-board computers
+				</p>
+			</Flex>
+
+		</Flex>
+
+		<MainSlider slideTiming={100000} />
+
+		<div className={styles.indicators}>
+			<Flex className={styles.homeContainer}>
+				<Flex className={styles.indicator} alignItems='center'>
+					<Flex
+						justifyContent='center'
+						alignItems='center'
+						className={styles.indicatorCircle}
+					>
+						<img className={styles.indicatorCircleImg} src={boxSVG} alt=''/>
+					</Flex>
+					<div className={styles.indicatorParameter}>
+						<p className={cn([styles.indicatorParameterTop, styles.text])}>5211</p>
+						<p className={cn([styles.indicatorParameterBottom, styles.text])}>Built images</p>
+					</div>
+				</Flex>
+				<Flex className={styles.indicator} alignItems='center'>
+					<Flex
+						justifyContent='center'
+						alignItems='center'
+						className={styles.indicatorCircle}
+					>
+						<img src={userSVG} className={styles.indicatorCircleImg}/>
+					</Flex>
+					<div className={styles.indicatorParameter}>
+						<p className={cn([styles.indicatorParameterTop, styles.text])}>2448</p>
+						<p className={cn([styles.indicatorParameterBottom, styles.text])}>Active users</p>
+					</div>
+				</Flex>
+			</Flex>
+		</div>
+
+		<div className={styles.smallSliderWrapper}>
+			<Flex className={styles.homeContainer}/>
+		</div>
+
+		<div className={styles.signUpToday}>
+			<Flex className={styles.homeContainer} justifyContent='center'>
+				<Flex
+					className={styles.signUpTodayContainer}
+					direction='column'
+					alignItems='center'
+				>
+					<Flex
+						className={styles.signUpTodayHeader}
+						justifyContent='center'
+						alignItems='center'
+					>
+						<img
+							className={styles.signUpTodayIcon}
+							src={shieldSVG}
+							alt=''
+						/>
+						<span>Sign Up Today</span>
+					</Flex>
+					<hr className={styles.signUpTodayHr}/>
+					<p className={styles.signUpTodayPhrase}>
+						Ultricies nec, pellentesque eu, pretium quis, sem.
+						Nulla consequat massa quis enim.
```

Comment:
wtf

Reply:
Do not review code, if you are not in the task.
I mean... that's a mock texts from designs, you should have knew that